addons/marketing_card/wizards/card_card_share.py

```diff
@@ -0,0 +1,46 @@
+from ast import literal_eval
+from markupsafe import Markup
+
+from odoo import _, fields, models
+
+
+class CardsCardShare(models.TransientModel):
+    _name = 'card.card.share'
+    _description = 'Email Marketing Cards Wizard'
+
+    def _default_message(self):
+        return Markup(_("""
+<p>Hello everyone,</p>
+
+<p>Here's the link to advertise your participation.<br>
+Your help with this promotion would be greatly appreciated!</p>
+
+<p>Many thanks</p>
+    """))
+
+    card_campaign_id = fields.Many2one('card.campaign', string='Marketing Card Campaign', ondelete='cascade', required=True)
+    domain = fields.Char(default="[]")
+    subject = fields.Char()
+    message = fields.Html(default=_default_message)
+    res_model = fields.Selection(related='card_campaign_id.res_model')
+
+    def action_send(self):
+        self.ensure_one()
+        target_records = self.env[self.res_model].search(literal_eval(self.domain) or [])
+        composer = self.env['mail.compose.message'].create({
+            'composition_mode': 'mass_mail',
+            'model': self.res_model,
+            'res_ids': target_records.ids,
+            'subject': self.subject,
+            'body': Markup("""
+                <div>{}</div>
+                <a t-att-href="env['card.campaign'].browse(ctx['marketing_card_campaign_id'])._get_preview_url(object.id)" class="o_no_link_popover">{}</a>
+            """).format(self.message or '', _("Your Card"))
```

Comment:
Strange, we have kind of the same use case for "survey.invite" but it doesn't use a composer (and uses a full blown mail.template):

![image](https://github.com/odoo/odoo/assets/41282536/7cd18ec2-9b81-426d-8bf6-42b61ff920b3)

Not necessarily saying you have to change it though, it's probably something you have already discussed with Thibault.